packages/core/src/codewhispererChat/tools/fsRead.ts

```diff
@@ -0,0 +1,175 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import * as vscode from 'vscode'
+import { getLogger } from '../../shared/logger/logger'
+
+// Constants
+// eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
Why do we need to disable this lint rule? It just expects consts to be declared in camal case

Reply:
Fixed.